src/v/security/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ v_cc_library(
   SRCS
     scram_algorithm.cc
     scram_authenticator.cc
+    gssapi_authenticator.cc
```

Comment:
i don't think this file has been added yet

Reply:
Good spot, fixed.